lib/file_locator.rb

```diff
@@ -51,7 +51,7 @@ def location
     when 's3'
       S3File.new(uri).object.presigned_url(:get)
     when 'file'
-      Addressable::URI.unencode(uri.path)
+      Addressable::URI.unencode(uri.to_s.gsub(/file:\/\//, ''))
```

Comment:
Could this also use `sanitize_uri`?
```suggestion
      Addressable::URI.unencode(ActiveEncode.sanitize_uri(uri))
```

Reply:
Good catch. Missed that one when consolidating everything else.